src/app/core/services/user/user.service.ts

```diff
@@ -33,8 +37,13 @@ interface CreateBusinessCustomerType extends Customer {
 
 @Injectable({ providedIn: 'root' })
 export class UserService {
-  constructor(private apiService: ApiService) {}
+  private isAppTypeREST: boolean;
+  private customerRestResource = 'customers';
 
+  constructor(private apiService: ApiService, store: Store) {
+    store.pipe(select(isAppTypeREST)).subscribe(isTypeREST => (this.isAppTypeREST = isTypeREST));
```

Comment:
I just tried using a facade and I see no issues there. What was the issue when you tried it?

Reply:
I got an error at runtime: cannot read property 'subscribe' of null

Reply:
I tried it without subscribe. As all the calls are RxJS, you can use it with `withLatestFrom` :wink: